src/config/types.d.ts

```diff
@@ -1,6 +1,45 @@
+type Environments =
+  | 'chrome'
+  | 'opera'
+  | 'edge'
+  | 'firefox'
+  | 'safari'
+  | 'ie'
```

Comment:
What is this nastyness

Reply:
This are possible values for `babel-env` config.